internal/cloudapi/v2/compose.go

```diff
@@ -1369,9 +1365,9 @@ func convertDiskCustomizations(disk *Disk) (*blueprint.DiskCustomization, error)
 				PartType: common.DerefOrDefault(fs.PartType),
 				MinSize:  fsSize,
 				FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
-					Mountpoint: fs.Mountpoint,
```

Comment:
Pardon my ignorance here - this is now a pointer because it is no longer marked "required" in the openapi spec so that auto-generated code makes it a pointer?